clients/csharp-uwp/UWPVoiceAssistantSample/KeywordRegistration.cs

```diff
@@ -35,20 +36,17 @@ public class KeywordRegistration : IKeywordRegistration, IDisposable
             string keywordId,
             string keywordModelId,
             string keywordActivationModelDataFormat,
-            string keywordActivationModelFilePath,
-            Version availableActivationKeywordModelVersion,
-            string confirmationKeywordModelPath)
+            Version availableActivationKeywordModelVersion)
         {
             this.KeywordDisplayName = keywordDisplayName;
             this.KeywordId = keywordId;
             this.KeywordModelId = keywordModelId;
             this.KeywordActivationModelDataFormat = keywordActivationModelDataFormat;
-            this.KeywordActivationModelFilePath = keywordActivationModelFilePath;
             this.AvailableActivationKeywordModelVersion = availableActivationKeywordModelVersion;
-            this.ConfirmationKeywordModelPath = confirmationKeywordModelPath;
 
             this.creatingKeywordConfigSemaphore = new SemaphoreSlim(1, 1);
 
+            LocalSettingsHelper.CopyConfigAndAssignValues().GetAwaiter();
```

Comment:
Let's move this to GetOrCreateKeywordConfigurationAsync and await it

Reply:
And surround it with an (if model path == null) so we only run it the first time this method is called

Reply:
Done